flex/engines/graph_db/app/server_app.cc

```diff
@@ -22,7 +22,7 @@ static uint32_t get_vertex_vid(const gs::ReadTransaction& txn, uint8_t label,
   uint32_t vid = std::numeric_limits<uint32_t>::max();
```

Comment:
`using oid_t = int64_t;` declared in `flex/storages/rt_mutable_graph/types.h` should be removed, and we shall ensure can remove all usage of oid_t is this PR?